src/proto/gps.proto

```diff
@@ -20,6 +20,9 @@ enum SampleType {
   TOTAL_DATA_TYPES = 13;
   POSITION = 14;
   LINEAR_VELOCITY = 15;
+  RGB_IMAGE_SIZE = 16;
+  DOMAIN_IMAGE = 17;
```

Comment:
Great. Yes - please add yourself to the contributor list before Sergey.

I'll take a look at the updated PR tonight.

On Mon, Apr 18, 2016 at 1:55 PM, Bradly Stadie <notifications@github.com>
wrote:

> In src/proto/gps.proto
> <https://github.com/cbfinn/gps/pull/15#discussion_r60132424>:
>
> > @@ -20,6 +20,9 @@ enum SampleType {
> >    TOTAL_DATA_TYPES = 13;
> >    POSITION = 14;
> >    LINEAR_VELOCITY = 15;
> > +  RGB_IMAGE_SIZE = 16;
> > +  DOMAIN_IMAGE = 17;
>
> Changed it to CONTEXT_IMAGE and put a comment next to it in the proto
> file.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cbfinn/gps/pull/15/files/7cb072155dd03779a3cdcaca47d1c59ef5ba1449#r60132424>
>